entrypoint.sh

```diff
@@ -47,6 +47,7 @@ aws() {
     [[ "${TAGS}" ]] && optionals+="-tags ${TAGS} " # Custom tags applied to pod vm
     [[ "${USE_PUBLIC_IP}" == "true" ]] && optionals+="-use-public-ip " # Use public IP for pod vm
     [[ "${ROOT_VOLUME_SIZE}" ]] && optionals+="-root-volume-size ${ROOT_VOLUME_SIZE} " # Specify root volume size for pod vm
+    [[ "${DISABLE_CLOUD_CONFIG}" == "true" ]] && optionals+="-disable-cloud-config "
```

Comment:
Currently disabling of cloud config is only for aws and azure. And since the option in the global sections are generic, moving cloud config configuration to global will be confusing imho. 
May be once we figure out how to implement it for other providers, especially ibmcloud I'll make this part of cloud config and move it under the common section. Does that sound ok ?